### PR TITLE
Music album

### PR DESCRIPTION
**Pull request by momotsow**
In this milestone

- Create MusicAlbum class in a separate .rb file.
- Create Genre class with an association to the Item class (in a separate .rb file).

### CODE REVIEW

File: music_album.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+  def initialize(name, publish_date, _on_spotify)
+    super(publish_date)
+    @id = Random.rand(1...1000)
+    @name = name
+    @on_spotify = on_spotify
+  end
+
+  def can_be_archived?
+    super && @on_spotify == true
```
Comment (yosefHesham) [inline music_album.rb:14]: great work !, can it just be `super && @on_spotify` ? what do you think 🤔

**Review by yosefHesham — CHANGES_REQUESTED** [review]
Hello @momotsow Great Work doing those classes 💪
 


**Highlights**  ⭐ 

- No linter errors
- Classes implemented very well
- First one to do his task 🏆

File: music_album.rb (hunk 1 above)
Comment (yosefHesham) [inline music_album.rb:14]: great work !, can it just be super && @on_spotify ? what do you think 🤔

**Review by vanelnw — APPROVED** [review]
# STATUS: MILESTONE APPROVED 👍 

Hi @momotsow , 
### Great job !!!

Your task is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Genre and Music_album classes implemented very well
:heavy_check_mark:  No linter errors
:heavy_check_mark:  Very descriptive and detailed PR

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**